config/initializers/okcomputer.rb

```diff
@@ -2,4 +2,10 @@
 OkComputer.mount_at = false
 
 OkComputer::Registry.register "solr", OkComputer::HttpCheck.new(Blacklight.default_index.connection.uri.to_s.sub(/\/$/, '') + "/admin/ping")
-OkComputer::Registry.register "downloads-cache", OkComputer::DirectoryCheck.new((Rails.root + "tmp/cache/downloads").to_s, true)
+
+if Settings.DOWNLOAD_PATH
+  OkComputer::Registry.register "downloads-cache", OkComputer::DirectoryCheck.new(Settings.DOWNLOAD_PATH, true)
```

Comment:
I'm a _weee_ bit nervous about this since we don't really have a way to test this in stage AFAICT.

Reply:
Could we edit the config on the server to test?

Reply:
Sure, would you suggest temporarily setting a directory on stage that is diff from the temp dir?

Prod's dir is in `/var/cache`, should we just use the same dir on stage (permanently or temporarily)?

Reply:
I'm not sure we want to set it to the same as prod.. just maybe even `/tmp/foo` and create that ?

Reply:
Okay, that seems to have worked on stage when I deployed this branch, configed a DOWNLOAD_PATH in tmp, and added the directory.